zap/src/test/resources/org/zaproxy/zap/spider/parser/html/MetaElementsSpiderHtmlParser.html

```diff
@@ -19,6 +19,18 @@
 <meta http-equiv="location" content="url=/meta/location/absolute">
 <meta http-equiv="location" content="url=ftp://meta.example.com/location">
 
+<meta http-equiv="Content-Security-Policy" content="script-src 'self'; report-uri /meta/csp">
+<meta http-equiv="Content-Security-Policy" content="script-src 'self'; report-uri //meta.example.com:4444/meta/base/csp/scheme">
+<meta http-equiv="Content-Security-Policy" content="script-src 'self'; report-uri https://meta.example.com/meta/csp">
+<meta http-equiv="Content-Security-Policy" content="script-src 'self'; report-uri meta/csp/refresh/relative">
+<meta http-equiv="Content-Security-Policy" content="script-src 'self'; report-uri ftp://meta.example.com/meta/csp/">
+
+<meta name="msapplication-config" content="/meta/msapplication">
+<meta name="msapplication-config" content="//meta.example.com:1337/meta/msapplication">
+<meta name="msapplication-config" content="https://meta.example.com/meta/msapplication">
+<meta name="msapplication-config" content="meta/msapplication/refresh/relative">
+<meta name="msapplication-config" content="ftp://meta.example.com/meta/msapplication/refresh">
```

Comment:
Missing the following cases:
 - Name empty.
 - Other name than `msapplication-config`.
 - Name `msapplication-config` and content `none` and empty.

Reply:
Added

Reply:
The case with content as `none` was not added.

Reply:
Added it to the `<!-- Ignored: -->` section.